### PR TITLE
everybody has goals

### PR DESCRIPTION
**Pull request by zendern**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -49,6 +51,9 @@ func handleEvent(ie slackevents.EventsAPIInnerEvent) {
 		if IsGritty(strings.ToLower(ev.Text)) {
 			_ = api.AddReaction("party-grit", slack.NewRefToMessage(ev.Channel, ev.TimeStamp))
 		}
+		if meetRavesGoal(ev) {
+			_ = api.AddReaction("middle_finger", slack.NewRefToMessage(ev.Channel, ev.TimeStamp))
```
Comment (fitzoh) [inline main.go]: is this a fireable offense?

Comment (zendern) [reply main.go]: we are going to find out :)

**Comment by fitzoh** [comment]
It feels weird giving real feedback on this, but...

The code is funny, but it's probably not gonna make a whole lot of sense when it triggers.

Maybe if 🖕 was added as :raves_goals: or something?

**Comment by zendern** [comment]
o yeah that makes it better :) hold please ill fix that

**Review by fitzoh — APPROVED** [review]
😂

File: main.go [hunk 2]
```diff
@@ -80,6 +85,14 @@ func hasAChickenWord(w *strset.Set) bool {
 	return !strset.Intersection(chickenWords, w).IsEmpty()
 }
 
+func meetRavesGoal(m *slackevents.MessageEvent) bool {
+	return m.User == ravesUserId && probablyNotGonnaHappenButMaybeCrazyJamieWillDoIt()
+}
+
+func probablyNotGonnaHappenButMaybeCrazyJamieWillDoIt() bool {
```
Comment (tylerrasor) [inline main.go:92]: we can hope!

**Pull request merged**